app/docs/reference/compatible-devices.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Compatible devices | Reference"
+description: "How do I know if my device is compatible with Routegy touchpoints?"
+sidebar: false
+---
+
+# Devices compatible with QR and NFC touchpoints
+
+Interacting with Routegy QR and NFC touchpoints relies on built-in capabilities of customer's device. These capabilities are now a standard feature in most modern iOS and Android devices.
+
+## iOS device compatibility
+
+QR code reading capability is built into the camera app on any iPhone, iPad and iPod that runs **iOS 11** or newer.
```

Comment:
The capability to scan QR codes is built into the camera app on iPhones, iPads, and iPods that run **iOS 11** or newer.